app/models/application_record.rb

```diff
@@ -19,8 +20,4 @@ class ApplicationRecord < ActiveRecord::Base
     extend MiqDecorator::Klass
     include MiqDecorator::Instance
   end
-
-  def self.display_name(number = 1)
-    n_(model_name.singular.titleize, model_name.plural.titleize, number)
```

Comment:
You do know that `self.display_name` is being overridden by many models in our application and you cannot just remove it from here and change its semantics completely, don't you?